lib/stringlib/lua/arith.lua

```diff
@@ -0,0 +1,87 @@
+local n = {}
+local meta = {}
+setmetatable(n, meta)
+meta.__tostring = function() return "<n>" end
+meta.__add = function(x, y)
+    if x == "BOOM" then
+        error(x)
+    end
+    return x
```

Comment:
Did you mean `x + y` here?

Reply:
No, I meant x :).  This is just a means to test the behaviour of the metamethod, not an attempt to do correct math!